builtin/providers/aws/resource_aws_spot_instance_request_test.go

```diff
@@ -35,6 +36,45 @@ func TestAccAWSSpotInstanceRequest_basic(t *testing.T) {
 	})
 }
 
+func TestAccAWSSpotInstanceRequest_vpc(t *testing.T) {
```

Comment:
A new acceptance test `TestAccAWSSpotInstanceRequest_vpc` that validates starting a spot instance in a VPC works